linkerd/app/src/env.rs

```diff
@@ -71,6 +72,8 @@ pub const ENV_OUTBOUND_ROUTER_MAX_IDLE_AGE: &str = "LINKERD2_PROXY_OUTBOUND_ROUT
 pub const ENV_INBOUND_MAX_IN_FLIGHT: &str = "LINKERD2_PROXY_INBOUND_MAX_IN_FLIGHT";
 pub const ENV_OUTBOUND_MAX_IN_FLIGHT: &str = "LINKERD2_PROXY_OUTBOUND_MAX_IN_FLIGHT";
 
+pub const ENV_LABELS_FILE_PATH: &str = "LINKERD2_PROXY_LABELS_FILE_PATH";
```

Comment:
Sorry, I think this is my last issue... Can we rename the env var/constants to indicate this is for tracing? And, seeing that OC calls these _attributes_, I'm inclined to use that terminology (i.e. to indicate that this is actually not specifically tied to k8s or prometheus labels). Perhaps `LINKERD2_PROXY_TRACE_ATTRIBUTES_PATH`

Reply:
No problem, happy to do it. Updated all the terminology from `labels` to `attributes`